src/Compiler/CompilerPasses.cpp

```diff
@@ -95,15 +95,14 @@ void addONNXToMLIRPasses(mlir::PassManager &pm, bool targetCPU) {
     // Statically add extra passes
     for (int i = 0; i < repeatOnnxTransform; i++) {
       pm.addPass(mlir::createCanonicalizerPass());
+      pm.addPass(onnx_mlir::createSimplifyShapeRelatedOpsPass());
+      pm.addPass(onnx_mlir::createRewriteONNXToONNXPass());
       pm.addNestedPass<func::FuncOp>(onnx_mlir::createShapeInferencePass());
       pm.addNestedPass<func::FuncOp>(
           onnx_mlir::createConstPropONNXToONNXPass());
     }
   }
 
-  // Simplify shape-related ops.
-  pm.addPass(onnx_mlir::createSimplifyShapeRelatedOpsPass());
-
```

Comment:
Move `SimplifyShapeRelatedOpsPass` pass into ONNXOpTransform pass so that `SimplifyShapeRelatedOpsPass` can be called repeatedly.